pkg/ovs/openflow/interfaces.go

```diff
@@ -170,8 +170,8 @@ type CTAction interface {
 	LoadToMark(value uint32) CTAction
 	LoadToLabelRange(value uint64, rng *Range) CTAction
 	MoveToLabel(fromName string, fromRng, labelRng *Range) CTAction
-	// NAT action is used if the packet is not committed into the conntrack zone, and is required to leverage the
-	// original NAT configurations.
+	// NAT action translates the packet being processed in the way the connection has been set up with an earlier,
```

Comment:
Remove "," after "earlier"?
Can not understand the sentence: "the packet being processed in the way the connection has been set up". Please rephrase.

Reply:
Sure, updated.